src/main/java/com/powsybl/openloadflow/network/impl/LfLoadImpl.java

```diff
@@ -73,6 +73,17 @@ public LfBus getBus() {
         return bus;
     }
 
+    @Override
+    public boolean isFictitious() {
```

Comment:
It is not so simple... the changes must be at lower level, when we compute the participation factor using `getAbsVariableTargetP()`. Note that this method is used at several places, for reach you must ask yourself if you need to change something or not. You also have to change the state update.

Reply:
And other remark, in iidm model, we have two ways to say that a load is fictitious (and I don't know why), through the type and through the boolean you are using. Please check both with a ||.